python/lsst/ts/mthexapod/hexapod_csc.py

```diff
@@ -677,6 +677,7 @@ def telemetry_callback(self, server):
         server : `lsst.ts.hexrotcomm.CommandTelemetryServer`
             TCP/IP server.
         """
+        tai_unix = server.header.tai_sec + server.header.tai_nsec / 1e9
```

Comment:
This is TAI time in unix format, right? Not Unix time like in here: https://en.wikipedia.org/wiki/Unix_time?

Reply:
Yes. TAI time in unix seconds, which is the T&S standard for all timestamps.